docs/pages/versions/v52.0.0/sdk/notifications.mdx

```diff
@@ -455,23 +467,25 @@ This value will be one of the following, accessible under `Notifications.IosAuth
+
+Several listeners are exposed and documented in the [Push notification behaviors](/push-notifications/what-you-need-to-know/#push-notification-behaviors) section.
+
+## Headless (Background) notifications
+
+See the [definition](/push-notifications/what-you-need-to-know/#headless-background-notifications) of Headless Background Notifications in the [What you need to know](/push-notifications/what-you-need-to-know) guide.
 
-A few different listeners are exposed, so we've provided a chart below which will hopefully help you understand when you can expect each one to be triggered:
+To handle notifications while the app is in the background or not running, you need to do the following:
 
-| User interacted with notification? | App state  | Listener(s) triggered                                                   |
-| ---------------------------------- | ---------- | ----------------------------------------------------------------------- |
-| false                              | Foreground | `NotificationReceivedListener` & `registerTaskAsync`                    |
-| false                              | Background | `registerTaskAsync`                                                     |
-| false                              | Killed     | `registerTaskAsync`                                                     |
-| true                               | Foreground | `NotificationReceivedListener` & `NotificationResponseReceivedListener` |
-| true                               | Background | `NotificationResponseReceivedListener`                                  |
-| true                               | Killed     | `useLastNotificationResponse` or `getLastNotificationResponseAsync`     |
+- Add `expo-task-manager` package to your project.
+- [Configure background notifications](#background-notification-configuration).
+- In your application code, set up a [background task](#registertaskasynctaskname) to run when the notification is received.
+- You must use a [development build](/develop/development-builds/introduction/) to use background notifications since they are not supported in Expo Go.
 
-In the table above, whenever `NotificationResponseReceivedListener` is triggered, the same would apply to the `useLastNotificationResponse` hook.
+Then send a push notification which:
 
-> **info** When the app is not running or killed and is restarted by tapping on a notification, the `NotificationResponseReceivedListener` may or may not be triggered. To reliably capture the response, we recommend using `useLastNotificationResponse` or `getLastNotificationResponseAsync`.
+- Contains only the `data` key (no `title`, `body`)
+Has `_contentAvailable: true` set for iOS &mdash; see the [Expo push notification service payload format](/push-notifications/sending-notifications/#message-request-format)
```

Comment:
```suggestion
- Has `_contentAvailable: true` set for iOS &mdash; see the [Expo push notification service payload format](/push-notifications/sending-notifications/#message-request-format)
```

Looks like my previous suggestion had a typo. Since this a list item, this will help resolve the CI error.